voluncheer/profiles/urls.py

```diff
@@ -2,11 +2,25 @@
 from django.urls import re_path
 
 from profiles.views.home import home
+from profiles.views.home import SignUpView
+from profiles.views.organizations import OrganizationSignUpView
 from profiles.views.profile import ProfileView
 from profiles.views.profile import profile_update
+from profiles.views.volunteers import VolunteerSignUpView
 
 urlpatterns = [
     path("", home, name="home"),
     path(r"^profile/$", ProfileView.as_view(), name="profile"),
```

Comment:
If you're cleaning this up please remove the regex anchors as they are not be necessary. Here and below.

Reply:
Same

Reply:
Done.